storage/quota/quota.go

```diff
@@ -320,3 +329,90 @@ func loadQuotaIDs(repquotaOpt string) (map[uint32]struct{}, uint32, error) {
 	logrus.Infof("Load repquota ids: %d, list: %v", len(quotaIDs), quotaIDs)
 	return quotaIDs, minID, nil
 }
+
+func getMountpoint(dir string) (string, error) {
+	var (
+		mountPoint string
+	)
+
+	output, err := ioutil.ReadFile(procMountFile)
+	if err != nil {
+		logrus.Warnf("failed to read file: (%s), err: (%v)", procMountFile, err)
+		return "", errors.Wrapf(err, "failed to read file: (%s)", procMountFile)
+	}
+
+	// /dev/sdb1 /home/pouch ext4 rw,relatime,prjquota,data=ordered 0 0
+	for _, line := range strings.Split(string(output), "\n") {
+		parts := strings.Split(line, " ")
+		if len(parts) != 6 {
+			continue
+		}
+
+		if strings.HasPrefix(dir, parts[1]) && len(parts[1]) > len(mountPoint) {
```

Comment:
What will happen if there are two reacords in `procMountFile` and the input dir name is /home/pouch? @rudyfly  
```
/dev/sdb1 /home/pouch ext4 rw,relatime,prjquota,data=ordered 0 0
/dev/sdb2 /home/pouch/a/b/c ext4 rw,relatime,prjquota,data=ordered 0 0
```

Reply:
I have checked `len(parts[1]) > len(mountPoint)`, It will choose the longest mount tab.

Reply:
Roger, How about adding this details in the comment of this function or the code block? @rudyfly 
Otherwise, I think there will be confusions there.

Reply:
ok